index.html

```diff
@@ -313,63 +317,73 @@ <h1>Get Started</h1>
-                        <br>
-                        <input type="text" id="fname" name="firstname" maxlength="30" required placeholder="Your name..">
-                    </div>
-                    <div class="email">
-                        <label for="email">E-mail</label>
-                        <br>               
-                        <input type="email" id="email" name="_replyto" required placeholder="Your e-mail..">
+            <div id="get-started" class="form">
+                <form action="https://formspree.io/f/myyldona" method="POST">
+                    <div class="formcontainer">
+                        <div class="fistname">
+                            <label for="fname">First Name</label>
+                            <br>
+                            <input type="text" id="fname" name="firstname" maxlength="30" required placeholder="Your name..">
+                        </div>
+                        <div class="email">
+                            <label for="email">E-mail</label>
+                            <br>               
+                            <input type="email" id="email" name="_replyto" required placeholder="Your e-mail..">
+                        </div>
                     </div>
-                </div>
-                <label for="textarea">Message</label>
-                <textarea name="message" id="textarea" cols="30" rows="10" maxlength="500" required placeholder="Your message here ..."></textarea>
-                <button class="btnsend" type="submit">Start Collaboration</button>
-            </form>
-            
-        </div>   
+                    <label for="textarea">Message</label>
+                    <textarea name="message" id="textarea" cols="30" rows="10" maxlength="500" required placeholder="Your message here ..."></textarea>
+                    <button class="btnsend" type="submit">Start Collaboration</button>
+                </form>
+
+            </div> 
+        </div>  
     </section>
 
     <footer class="mobile-footer">
-        <button class="mobile-btncv">Get my CV <img src="images/ic_link_teal.svg" alt=""></button> 
-    <div class="mobile-media-share">
-        <a href="http://stackoverflow.com"><img src="images/ic_stackoverflow.svg" alt=""></a>
-        <a href="https://github.com/leonardoeng13"><img src="images/ic_github.svg" alt=""></a>
-        <a href="https://twitter.com/andrede_leo"><img src="images/ic_twitter.svg" alt=""></a>
-        <a href="https://www.linkedin.com/in/leonardodeandrade"><img src="images/ic_linkedin.svg" alt=""></a>
-        <a href="https://medium.com/"><img src="images/ic_medium.svg" alt=""></a>
-        <a href="https://angel.co/"><img src="images/ic_angellist.svg" alt=""></a>
-    </div>
+        <form action="cv/Leonardo_Andrade_English_2021.pdf" method="get">
+            <button class="mobile-btncv" type="submit">Get my CV<img src="images/ic_link_teal.svg" alt=""></button>
+          </form>
```

Comment:
- [ ] Check all your buttons to implement the following:
* Add hover color
* Modify your the button as cursor:pointer

Reply:
At this point, I'm using my own implementations. These requirements you're requiring changes are for next week.